components/brave_wallet/browser/fil_response_parser_unittest.cc

```diff
@@ -180,4 +188,79 @@ TEST(FilResponseParserUnitTest, ParseFilEstimateGas) {
                                                  &gas_fee_cap, &gas_limit));
 }
 
+TEST(FilResponseParserUnitTest, ParseFilGetChainHead) {
```

Comment:
Can we add a case for height being an empty string?

Reply:
done